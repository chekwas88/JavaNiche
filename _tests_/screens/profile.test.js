import React from 'react';
import { shallow } from 'enzyme';
import ProfileScreen from '../../src/screens/Profile';

describe('Test for RepoButton component', () => {
  it('should render correctly', () => {
    const props = {
      navigation: {
        navigate: jest.fn(),
        state: {
          params: {
            email: '',
            fullName: '',
            userName: '',
            avatarUrl: '',
            repos: '',
            starredRepos: '',
            reposUrl: ''
          }
        }
      }
    };
    const wrapper = shallow(<ProfileScreen {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
