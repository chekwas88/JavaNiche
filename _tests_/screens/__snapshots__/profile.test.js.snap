// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for profile screen should render correctly 1`] = `ShallowWrapper {}`;
